astropy/wcs/tests/test_wcs.py

```diff
@@ -793,10 +793,14 @@ def test_error_message():
     hdr["PV1_2"] = 110
     hdr["PV2_1"] = -110
     hdr["PV2_2"] = -110
+
+    def init_wcs_and_alter(hdr):
+        w = wcs.WCS(hdr, _do_set=False)
+        w.all_pix2world([[536.0, 894.0]], 0)
```

Comment:
Is the helper function needed?

Reply:
No just had to flip flop the with statements.